src/sentry/tasks/sentry_apps.py

```diff
@@ -297,16 +297,34 @@ def send_and_save_webhook_request(url, sentry_app, app_platform_event):
     org_id = app_platform_event.install.organization_id
     event = "{}.{}".format(app_platform_event.resource, app_platform_event.action)
 
+    slug = sentry_app.slug
+    if sentry_app.status == 2:
+        slug = "internal"
+    if sentry_app.status == 0:
+        slug = "unpublished"
```

Comment:
I have no idea, but I feel like this would be a good thing to `@property` onto the actual SentryApp model IMO, since it's kinda useful

Reply:
Agreed

Reply:
you can use `sentry_app.is_published`, `sentry_app.is_unpublished`, and `sentry_app.is_internal`

Reply:
but also yeah since you are repeating code a lot, could add a property (like leander said) that was like `slug_for_metrics` iono